test/suites/tuned/workload-partitioning.robot

```diff
@@ -0,0 +1,258 @@
+*** Settings ***
+Documentation       Tests for Workload partitioning
+
+Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/common.resource
+Resource            ../../resources/systemd.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/microshift-network.resource
+
+Suite Setup         Setup Suite With Namespace
```

Comment:
Looks like previous test (tuned) does not wait for MicroShift readiness right after rebooting in the teardown. This can make this test run too fast and MicroShift is not available yet.
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_microshift/3700/pull-ci-openshift-microshift-main-microshift-metal-tests/1821188298421309440/artifacts/microshift-metal-tests/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/el94-src@low-latency/log.html
Tuned teardown: `Start / End / Elapsed:	20240807 17:13:21.895 / 20240807 17:13:58.041 / 00:00:36.146`
workload partitioning setup trying to create a namespace: `Start / End / Elapsed:	20240807 17:13:59.543 / 20240807 17:13:59.652 / 00:00:00.109`
This failed because MicroShift is not up yet.

Reply:
yes, we already fixed that in the other PR, cherry picking the change.